ui/jobs/jobs.js

```diff
@@ -3124,6 +3226,48 @@ class Bars {
           this.buffTracker.onUseAbility(m.groups.id, m.groups);
       }
     }
+    // For extremely complex BRD
+    if (this.job != 'BRD') return;
```

Comment:
Can you put return on its own line? This is just my own personal style, but I find it very easy to overlook a `return` at the end of the line.